assembl/static/js/app/routeManager.js

```diff
@@ -408,19 +412,47 @@ var routeManager = Marionette.Object.extend({
           });
       });
     widgetPromise.then(function(widget){
-      var CurrentWidgetView;
+      if (!widget){
+        var error_text = "Requested vote widget could not be found.";
+        alert(error_text);
```

Comment:
Why `alert` in a prod system? Should instead show the error `growl` instead and not show the modal/view instead.

Reply:
`error_text` should also be i18n'ed, and turned into `camelCase`